Sources/EmbraceSemantics/SpanEvent/SpanEventSemantics+Breadcrumb.swift

```diff
@@ -9,7 +9,7 @@ public extension SpanType {
 }
 
 public extension SpanEventSemantics {
-    struct Bradcrumb {
+    struct Breadcrumb {
         public static let name = "emb-breadcrumb"
         public static let keyMessage = "message"
     }
```

Comment:
Sadly, `EmbraceSemantics` is a public target, and is meant to be used by clients; to prevent any kind of unnecessary migration, i'd do something like this:
```swift
public extension SpanEventSemantics {
    @available(*, deprecated, message: "Use Breadcrumb as this struct will be removed in future versions", renamed: "Breadcrumb")
    struct Bradcrumb {
        public static let name: String = Breadcrumb.name
        public static let keyMessage: String = Breadcrumb.keyMessage
    }

    struct Breadcrumb {
        public static let name = "emb-breadcrumb"
        public static let keyMessage = "message"
    }
}
```

Reply:
Ah interesting good to know! Would this also affect the other usages of this in Sources/EmbraceCore/Public/Events/Breadcrumb.swift ?

Reply:
Shouldn't, because you already made the modification of those, and it should behave the same (it's just a change in the name of the `struct`